test/snippet/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+# ============================================================================
```

Comment:
this file needs to be added in an extra PR and commit. I thought @marehr was supposed to do this?